.circleci/config.yml

```diff
@@ -409,9 +409,15 @@ jobs:
 
   deploy-dev:
     executor: docker-executor
+    parameters:
```

Comment:
By accepting parameters here we only need to define this job once and we can target all dev environments.